### DISCUSSION TITLE
Remove firmware build from build.sh?

### DISCUSSION THREAD

**Discussion by jpreiss** [USC-ACTLab/crazyswarm#391] [Ideas]
Most Crazyswarm users won't need to modify the firmware (I think?), but `build.sh` will fail unless the user has the ARM cross-compiler set up correctly (see e.g. https://github.com/USC-ACTLab/crazyswarm/discussions/390).

I believe the firmware build in `build.sh` is an artifact from when we had a custom fork of the firmware. We could remove it.

We still need to include the firmware as a submodule for the python bindings, but in most cases users should never need to build the full ARM binary.

**Comment by whoenig** [comment]
Agreed. We can also remove the NRF firmware submodule and potentially the one for the radio (the radios still ship with the older firmware by default, so upgrading is mandatory, but it is the latest official radio firmware at least).
